chaoscenter/web/src/views/AccountSettingsUserManagement/AccountSettingsUserManagement.tsx

```diff
@@ -230,16 +244,35 @@ export default function AccountSettingsUserManagementView(
           small
           loading={useUsersQueryLoading}
           noData={{
-            when: () => !usersData?.length,
+            when: () => !currentItems?.length,
             message: getString('noUserAddUsers')
           }}
         >
-          <Text font={{ variation: FontVariation.H4 }}>
-            {getString('totalUsers')}: {usersData?.length ?? 0}
-          </Text>
+          <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+            <Text font={{ variation: FontVariation.H4 }}>
+              {getString('totalUsers')}: {usersData?.length ?? 0}
+            </Text>
+            <div>
+              Items per page:
+              <select value={itemsPerPage} onChange={handleItemsPerPageChange}>
+                <option value={5}>5</option>
+                <option value={10}>10</option>
+                <option value={20}>20</option>
+              </select>
+            </div>
+          </div>
```

Comment:
This feature is already present in pagination component and need not be re-written. Refer to the comment below.